app/models/concerns/hyrax/has_rendering.rb

```diff
@@ -0,0 +1,12 @@
+module Hyrax::HasRendering
+  extend ActiveSupport::Concern
+
+  included do
+    # rubocop:disable Rails/HasAndBelongsToMany
+    # dc:hasFormat defined in Presentation API context:  http://iiif.io/api/presentation/2/context.json
+    has_and_belongs_to_many :renderings,
+                            predicate: ::RDF::Vocab::DC.hasFormat,
```

Comment:
Will using this predicate be problematic? @no-reply (Tagging you since you started the work to make the Admin Set relationship predicate configurable.)

Reply:
I guess it could be, but this is the *proper* predicate according to IIIF. We could make it configurable ... or leave as-is until Hyrax 3 when changing predicates will get easier.

Reply:
:ok: I'd like to hear from @no-reply on this too.

Reply:
Configurable would be great. It doesn't look like there's any particular reason a schema for a Hyrax Work would need to align with the IIIF schema, so I'm not sure we care what they say is the proper predicate.

It does suggest the kinds of conflicts--with a specialized use of a common DC predicate--that we saw with the partitive predicates. I would vote for:

  - Follow IIIF by default and use `hasFormat`, but make it configurable; or
  - Use something internal (`something:iiifRenderings`) and serialize to IIIF manifiests as appropriate (and maybe make it configurable either now later?).

I don't have strong opinions about which.

Reply:
@geekscruff I'm fine with either of these approaches. Clear?

Reply:
Yep, sounds good to me. I'll try to get this wrapped up tomorrow.

Reply:
I've gone with option 1.